### DISCUSSION TITLE
How should I deal with deprecated ACF field types

### DISCUSSION THREAD

**Discussion by hirasso** [qtranslate/qtranslate-xt#1351] [Q&A] [answered]
Hi there! I'm maintaining dozens of legacy websites and got a bit worried when I noticed that the ACF field types are now deprecated. I'm making use of almost all of them:

- File [qTranslate-XT]
- Image [qTranslate-XT]
- Post Object [qTranslate-XT]
- Text [qTranslate-XT] - Deprecated
- Text Area [qTranslate-XT] - Deprecated
- URL [qTranslate-XT]
- WYSIWYG Editor [qTranslate-XT] - Deprecated

I couldn't find any information about how to deal with those. Could you provide some information about the official recommended upgrade path? 

- Will the deprecated fields continue to work, or 
- should sites migrate them to native ACF fields? If so, would that cause loosing the translated values?

**Comment by hirasso** [comment]
Is there any official documentation available that I'm not aware of? @herrvigg

**Comment by herrvigg** [comment, marked as answer]
This come from the changes I made in pull request #1305.
> Since the standard ACF fields are better supported, those should be preferred over the qTranslate extended fields. 
> In particular: text, textarea, wysiwyg are covered by the ACF standard fields, while others only exist as qTranslate extended fields.

The standard fields should give a better support on the long run.
We should only use specific QTX custom fields when it's strictly necessary.
Does it answer your question?

**Reply by hirasso** [reply]
Yes, thank you!

**Reply by ntd96** [reply]
hi sir, if I use standard fields, when I delete the acf-qtranslate plugin, I now have to manually switch back to basic standard fields, while there are hundreds of fields, is there a faster way?

**Reply by herrvigg** [reply]
There should be a faster way. I haven't tried myself, but the raw content should be the same. So it's just about changing the field type, in theory.
